rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= RPA Bot 1.4.0 Release Notes
+
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== November , 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The script now provides the Bot GUID when executing a workflow so that the Bot GUID can be provided via runtime variable in RPA Builder.
```

Comment:
"via runtime ..." -> "via the..." or "via a runtime"?

Reply:
Do the first 2 bullets need to be parallel to the last ones that start with the verb?